src/LoggerBase.php

```diff
@@ -16,27 +18,36 @@ abstract class LoggerBase extends \Psr\Log\AbstractLogger
     /**
      * Interpolate context values into the message placeholders.
      * 
-     * @param string|array $message The data to be logged.
+     * @param string $message The data to be logged.
      * @param array $context (Optional:) The array of values to insert into the
      *     corresponding placeholders.
-     * @return mixed The resulting log message.
+     * @return string The resulting log message.
      */
-    protected function interpolate($message, array $context = [])
+    protected function interpolate($message, array $context = []): string
     {
         if (is_string($message)) {
             return $this->interpolateString($message, $context);
         }
         
-        if (is_array($message) && $this->isArraySupported) {
```

Comment:
Since we're removing the general support for array messages, we can remove this `isArraySupported` property from  this LoggerBase class.

Reply:
Done, not pushed yet.